src/parse/asp/objects.go

```diff
@@ -822,20 +822,40 @@ type pyConfig struct {
 
 func (c *pyConfig) MarshalJSON() ([]byte, error) {
 	if c.overlay == nil {
-		return json.Marshal(c.overlay)
+		return json.Marshal(c.base.dict)
 	}
+
+	return json.Marshal(c.Config())
+}
+
+func (c *pyConfig) Config() pyDict {
```

Comment:
(nit: needs a doc comment)

but does this need to be public? Maybe make it private and re-name it `toPyDict()`?